api/org/commcare/api/persistence/SqlSandboxUtils.java

```diff
@@ -13,13 +13,28 @@
  * @author wspride
  */
 public class SqlSandboxUtils {
+
+    public static UserSqlSandbox getStaticStorage(String username, String path) {
```

Comment:
Did you include this to write a new test to fix the bug? If so is that test included anywhere? There's new code here, but no new code paths that use it